app/main.py

```diff
@@ -2,4 +2,63 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self, name: str, weight: int, coords: list[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
```

Comment:
I think  better use `or`
```suggestion
        self.coords = coords or [0, 0]
```